### PR TITLE
feature: add new language (lt-LT)

### PR DESCRIPTION
**Pull request by AndreyNegrov**
add new language (lt-LT)

### CODE REVIEW

File: locales/lt-LT/messages.po [hunk 1]
```diff
@@ -1,7 +1,7 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: TAO 3.3.0-sprint96\n"
-"PO-Revision-Date: 2019-04-25T01:14:03\n"
+"Project-Id-Version: TAO 3.4.0-sprint112\n"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Are you sure about switching Tao version frmo 3.3.0 to 3.4.0?

File: locales/lt-LT/role.rdf.po [hunk 2]
```diff
@@ -1,7 +1,7 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: TAO 3.3.0-sprint96\n"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/role.rdf.po]: Are you sure about switching Tao version frmo 3.3.0 to 3.4.0?

File: locales/lt-LT/messages.po (hunk 1 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/role.rdf.po (hunk 2 above)
Comment (AndreyNegrov) [reply locales/lt-LT/role.rdf.po]: fixed

**Comment by tikhanovichA** [comment]
I think we don't need that PR. Because you didn't change any translations for this extension.

**Pull request closed without merging**